javascript-modal/javascript-modal/styles.css

```diff
@@ -0,0 +1,48 @@
+body {
+  margin: 0;
+  font-family: sans-serif;
+}
+
+button {
+  font-size: 18px;
+  color: white;
+  padding: 10px 20px;
+  margin: 10px 10px;
+  box-shadow: 0 0 10px 0 grey;
+  border: none;
+  border-radius: 5px;
+  outline: none;
+}
+
+button.one {
+  background-color: royalblue;
+}
+
+button.two {
+  background-color: red;
+}
+
+.modal {
+  padding: 30px 20px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  position: fixed;
+  top: 180px;
```

Comment:
As of right now, this modal is very manually placed with these pixel values for `top` and `left`, which means it will only look centered on a very exact screen size. Is there any way you can get it centered on any viewport size?